src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAroundCheck.java

```diff
@@ -213,6 +213,23 @@
  * <pre>
  * &lt;module name=&quot;WhitespaceAround&quot;/&gt;
  * </pre>
+ * <p>Example:
+ * </p>
+ * <pre>
+ * class Test {
+ *     public Test(){}//..violation
```

Comment:
Please format your comment properly: `public Test(){}//..violation` to `public Test(){} // violation` (see https://checkstyle.sourceforge.io/config_naming.html#TypeName as an example). The same for your other changes.

Reply:
Done @rdiachenko

Reply:
@harsh-kukreja please don't resolve comments yourself. This one is still not done. Compare a string from my above comment: `public Test(){} // violation` and your changes `public Test(){}// violation`.

Reply:
@rdiachenko Sorry for that. Done with the changes please check.